src/redux/sagas/motions/moveFundsMotion.ts

```diff
@@ -112,65 +89,192 @@ function* moveFundsMotion({
         'annotateMoveFundsMotion',
       ]);
 
-    const [fromPermissionDomainId, fromChildSkillIndex] = yield call(
-      getPermissionProofs,
-      colonyClient.networkClient,
-      colonyClient,
-      fromDomain.nativeId,
-      [ColonyRole.Funding],
-      votingReputationClient.address,
-    );
+    // eslint-disable-next-line no-inner-declarations
+    function* getCreateMotionParams() {
+      const isOldVersion = colonyVersion <= 6;
+      const contractMethod = isOldVersion
+        ? 'moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,address)'
+        : 'moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,uint256,uint256,address)';
```

Comment:
Don't we have some enum types for these method signatures ?

Reply:
Not from what I can see! All the sagas currently have them as strings. I do feel like we should though 🤔